luigi/server.py

```diff
@@ -175,7 +175,7 @@ def initialize(self, scheduler):
         self._scheduler = scheduler
 
     def get_template_path(self):
-        return pkg_resources.resource_filename(__name__, 'templates')
+        return importlib.resources.files("templates").name
```

Comment:
@dlstadther this change caused coverage to fail. Not sure why this is happening since this is an existing function. Do you know where would be a good place to add a test for this?

Reply:
I just saw your other comment.

Reply:
The tests for `server.py` live in https://github.com/spotify/luigi/blob/master/test/server_test.py . It looks like multiple of these "Handler" classes are currently untested.